### DISCUSSION TITLE
Error on Generate LSM OL Cumulative Density Function

### DISCUSSION THREAD

**Discussion by Realone24** [NASA-LIS/LISF#1104] [Q&A] [answered]
Dear all,

I was working with available public test-case and modified it for different domain, But it was throwing error while reading LSM output files (STEP 4). However all the steps of test case runs were fine and produced desired output. 

The error message is below, please let me know how to overcome this issue.  

I have also checked the file variables and SoilMoist_tavg variable was there, I have attached all the relevant files here.

Thanks

[INFO] LDT cycle time: 04/02/2017 00:00:00
 [INFO] reading LSM output
 ../OL_OUTPUT//SURFACEMODEL/201704/LIS_HIST_201704020000.d01.nc
 [ERR] Error in nf90_get_var: SoilMoist_tavg Stopping.         -57
 endrun is being called

[file_vars.txt](https://github.com/NASA-LIS/LISF/files/8798461/file_vars.txt)
[ldtlog.0000.txt](https://github.com/NASA-LIS/LISF/files/8798462/ldtlog.0000.txt)
[ldt.config.noah36_cdf.txt](https://github.com/NASA-LIS/LISF/files/8798463/ldt.config.noah36_cdf.txt)

**Comment by Gayathridevi123** [comment, marked as answer]
Hi @Realone24,

I believe that this is happening because of your **Observation count threshold value**. As you have given 'Observation count threshold:          30', but your simulation period is small, like 4 days. So you may have to reduce this Observation count threshold value. You can refer about this value from the previous discussions https://github.com/NASA-LIS/LISF/discussions/840 and https://github.com/NASA-LIS/LISF/discussions/1071.  Hope this will help!

Thanks
Gayathri

**Reply by Realone24** [reply]
Hi @Gayathridevi123 

Thanks for the reply. I was giving wrong domain info for soil, which was causing the error. However, you have also highlighted a good point for improvement.

Thanks
Amit
